XRTK-Core/Packages/com.xrtk.core/Runtime/Providers/Controllers/Hands/HandDataPostProcessor.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using XRTK.Definitions.Controllers.Hands;
+using XRTK.Definitions.Devices;
+using XRTK.Definitions.Utilities;
+using XRTK.Services;
+using XRTK.Utilities;
+
+namespace XRTK.Providers.Controllers.Hands
+{
+    /// <summary>
+    /// The hand data post processor updates <see cref="HandData"/> provided
+    /// by a platform and enriches it with potentially missing information.
+    /// </summary>
+    public sealed class HandDataPostProcessor
```

Comment:
Any particular reason for these are sealed?

Reply:
They are not designed for inheritance.